test/multireddit/multireddit_test.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2018, the Dart Reddit API Wrapper project authors.
+// Please see the AUTHORS file for details. All rights reserved.
+// Use of this source code is governed by a BSD-style license that
+// can be found in the LICENSE file.
+
+import 'dart:async';
+import 'package:draw/draw.dart';
+import 'package:test/test.dart';
+import 'package:color/color.dart';
+
+import '../test_utils.dart';
+
+Future main() async {
+  final expectedSubredditList = [
+    {"name": "solotravel"},
+    {"name": "Fishing"},
+    {"name": "geocaching"},
+    {"name": "Yosemite"},
+    {"name": "CampingandHiking"},
+    {"name": "whitewater"},
+    {"name": "remoteplaces"},
+    {"name": "Adirondacks"},
+    {"name": "caving"},
+    {"name": "travelphotos"},
+    {"name": "canyoneering"},
+    {"name": "Backcountry"},
+    {"name": "Bushcraft"},
+    {"name": "Kayaking"},
+    {"name": "scuba"},
+    {"name": "bouldering"},
+    {"name": "CampingGear"},
+    {"name": "urbanexploration"},
+    {"name": "WWOOF"},
+    {"name": "adventures"},
+    {"name": "climbing"},
+    {"name": "Mountaineering"},
+    {"name": "sailing"},
+    {"name": "hiking"},
+    {"name": "iceclimbing"},
+    {"name": "tradclimbing"},
+    {"name": "backpacking"},
+    {"name": "trailmeals"},
+    {"name": "Climbingvids"},
+    {"name": "camping"},
+    {"name": "climbharder"},
+    {"name": "skiing"},
+    {"name": "Outdoors"},
+    {"name": "alpinism"},
+    {"name": "PacificCrestTrail"},
+    {"name": "Ultralight"},
+    {"name": "Hammocks"},
+    {"name": "AppalachianTrail"}
+  ];
+
+  test('lib/multireddit/copy', () async {
+    final reddit = await createRedditTestInstance(
+        'test/multireddit/lib_multireddit_copy.json');
+    final data = {
+      "kind": "LabeledMulti",
```

Comment:
nit: in general, use single quotes when possible (if you have single quotes in a string, that's an exception).

Reply:
Done.